google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -453,6 +468,21 @@ def autoclass_enabled= toggle
           patch_gapi! :autoclass
         end
 
+        ##
+        # Update method to update all attributes of autoclass of a bucket
+        # It accepts params as a Hash of attributes in the following format
+        #   { enabled: true, terminal_storage_class: "ARCHIVE" }
```

Comment:
4 space indent, not 2. (This is Markdown.) And probably best to have a blank line before it.

Reply:
I promise I thought I had done 2 tabs (2 space each) but it looks like I had done only 2 spaces. Fixed now.